### PR TITLE
Update project name

### PR DESCRIPTION
**Pull request by gbraad**
... and update `crc` reference

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,11 +23,11 @@ It can connect to multiple engines at the same time and provides an unified inte
 
 Specific runtime plug-ins provides support for different container engine.
 
-There are plug-ins to handle [Podman container engine](https://github.com/containers/podman), [CodeReady container](https://github.com/code-ready/crc), [Podman Lima machines](https://github.com/lima-vm/lima), Docker, etc.
+There are plug-ins to handle [Podman container engine](https://github.com/containers/podman), [crc](https://github.com/code-ready/crc), [Podman Lima machines](https://github.com/lima-vm/lima), Docker, etc.
```
Comment (benoitf) [inline README.md:26]: I would say we would need the 'full name' and not the acronym for the name.

**Comment by benoitf** [comment]
FYI DCO and Semantic check are failing (may prefix by docs: the commit)

File: README.md (hunk 1 above)
Comment (gbraad) [reply README.md:26]: the new name is "OpenShift Local" for the downstream, but only 'crc', with no direct meaning for upstream.

We could choose `crc/OpenShift Local` or `crc with OpenShift Local preset` as the name, WDTY?

Comment (benoitf) [reply README.md:26]: but there https://github.com/code-ready/crc it's written `Container Runtime Control` and `CodeReady Containers` everywhere

Comment (gbraad) [reply README.md:26]: that will change in the next weeks; (name change, and the removal of the CodeReady brand): https://github.com/code-ready/crc/pull/3068/files

**Review by benoitf — APPROVED** [review]

**Pull request merged**